### PR TITLE
fix: ml detector buffer capacity and parameterization

### PR DESCRIPTION
**Pull request by knzo25**
## Description
In some vehicle setups, the current 1M points was not enough in the ML object detectors.
In this PR I increased it to 2M while also making it a parameter so that users can adapt it to their use cases

## Related links
- [TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/RT0-31777)
- [TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/RT1-7150)

## How was this PR tested?
Launcher autoware and verified that the parameter was loaded correctly

## Notes for reviewers
This PR must be merged with its `autoware_launch` companion (https://github.com/autowarefoundation/autoware_launch/pull/1067)
None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: perception/lidar_centerpoint/schema/centerpoint.schemal.json [hunk 1]
```diff
@@ -12,6 +12,12 @@
           "default": "fp16",
           "enum": ["fp32", "fp16"]
         },
+        "cloud_capacity": {
+          "type": "integer",
+          "description": "Capacity of the point cloud buffer (should be set to at least the maximum theoretical amount of points).",
```
Comment (amadeuszsz) [inline perception/lidar_centerpoint/schema/centerpoint.schemal.json:17]: "number of points"

File: perception/lidar_transfusion/schema/transfusion.schema.json [hunk 2]
```diff
@@ -21,6 +21,12 @@
           "default": "fp16",
           "enum": ["fp16", "fp32"]
         },
+        "cloud_capacity": {
+          "type": "integer",
+          "description": "Capacity of the point cloud buffer (should be set to at least the maximum theoretical amount of points).",
```
Comment (amadeuszsz) [inline perception/lidar_transfusion/schema/transfusion.schema.json:26]: "number of points"

**Review by amadeuszsz — CHANGES_REQUESTED** [review]
Thanks for your PR! Apart of tiny typos it looks good!

File: perception/lidar_transfusion/schema/transfusion.schema.json (hunk 2 above)
Comment (knzo25) [reply perception/lidar_transfusion/schema/transfusion.schema.json]: Addressed in [fa1abeb](https://github.com/autowarefoundation/autoware.universe/pull/7936/commits/fa1abebd498d8d134c408c2c9fa92b890581ca96)

File: perception/lidar_centerpoint/schema/centerpoint.schemal.json (hunk 1 above)
Comment (knzo25) [reply perception/lidar_centerpoint/schema/centerpoint.schemal.json]: Addressed in [fa1abeb](https://github.com/autowarefoundation/autoware.universe/pull/7936/commits/fa1abebd498d8d134c408c2c9fa92b890581ca96)

**Review by amadeuszsz — APPROVED** [review]

**Comment by knzo25** [comment]
@kminoda @tzhong518 
friendly ping

**Review by kminoda — APPROVED** [review]
LGTM

**Pull request merged**